test/org.jkiss.dbeaver.test.platform/src/org/jkiss/dbeaver/model/sql/format/tokenized/SQLFormatterTokenizedTest.java

```diff
@@ -551,4 +552,16 @@ public void shouldDoDefaultFormatForSimpleSelectAndCaseEndConditionInOrderBlock(
         //then
         assertEquals(expString, formattedString);
     }
+
+    @Test
+    public void shouldCorrectlyHandleBackslashEscapedQuotesInStringLiterals() {
+        //given
```

Comment:
do we really need such comments?

Reply:
I dunno, it's a copypasta from tests above